resolver/resolver.go

```diff
@@ -68,6 +68,14 @@ func (resolver *Resolver) QueryStat(ctx context.Context, id interface{}, params
 	return resolver.handleMessage(ctx, id, jsonrpc.MethodQueryStat, *params, req, false)
 }
 
+func (resolver *Resolver) QueryTxs(ctx context.Context, id interface{}, params *jsonrpc.ParamsQueryTxs, r *http.Request) jsonrpc.Response {
+	// TODO : Implement the queryTx method.
```

Comment:
```suggestion
	// TODO: Implement the queryTx method.
```

Just removing the space so VSCode highlights it